pkgs/servers/ldap/lldap/default.nix

```diff
@@ -20,28 +20,31 @@ let
 
   commonDerivationAttrs = rec {
     pname = "lldap";
-    version = "0.5.0";
+    version = "0.5.1-alpha-6aa930";
 
     src = fetchFromGitHub {
       owner = "lldap";
       repo = "lldap";
-      rev = "v${version}";
-      hash = "sha256-2MEfwppkS9l3iHPNlkJB4tJnma0xMi0AckLv6wpzy1Y=";
+      # rev = "v${version}";
+      # hash = "";
+      rev = "6aa9303339ac1793b4be9ae01c56a6369a5d77f0";
+      hash = "sha256-UbkpnPvDoTaioB0g1HBvRczqUEuPlVuH1gIVBU07RpY=";
     };
 
+  };
+
+  frontend = rustPlatform.buildRustPackage (commonDerivationAttrs // {
+    pname = commonDerivationAttrs.pname + "-frontend";
+
     # `Cargo.lock` has git dependencies, meaning can't use `cargoHash`
     cargoLock = {
-      lockFile = ./Cargo.lock;
+      lockFile = ./Cargo.original.lock;
```

Comment:
Can't we use the same cargo.lock everywhere?

Reply:
I thought I did explain this in the summary, no?
Though it seems @alyssais found a way ...